.github/workflows/ci-preview.yml

```diff
@@ -0,0 +1,29 @@
+name: Generate Preview (CI)
+
+concurrency:
+  group: ci-${{ github.event.number }}
+  cancel-in-progress: true
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-preview:
+    if: ${{ github.repository_owner == 'cryostatio' }}
+    env:
+      DEPLOY_DOMAIN: ${{ github.repository_owner }}-cryostat-web-main.surge.sh
```

Comment:
`{{ github.event.repository.name }}` should work too instead of hardcoding the repo name, I think

Reply:
https://stackoverflow.com/questions/62803531/repository-name-as-a-github-action-environment-variable